### PR TITLE
Updated LDBC SNB Enhancer version

### PR DESCRIPTION
**Pull request by Sindhu-Vasireddy**

### CODE REVIEW

File: lib/Generator.ts [hunk 1]
```diff
@@ -97,6 +97,7 @@ export class Generator {
     // Create params.ini file
     const paramsTemplate = await fs.promises.readFile(Path.join(__dirname, '../templates/params.ini'), 'utf8');
     const paramsPath = Path.join(this.cwd, 'params.ini');
+    // eslint-disable-next-line unicorn/prefer-string-replace-all
```
Comment (rubensworks) [inline lib/Generator.ts]: Perhaps we can just apply the suggested linter enchancement :-)

File: templates/enhancer-config-pod.json [hunk 2]
```diff
@@ -26,4 +26,4 @@
       "@type": "EnhancementHandlerVocabulary"
     }
   ]
-}
+}
```
Comment (rubensworks) [inline templates/enhancer-config-pod.json]: There's a missing newline at the end of this file. Could you add one? Then git will just see this as a file rename, without differences.

**Review by rubensworks — CHANGES_REQUESTED** [review]

**Comment by Sindhu-Vasireddy** [comment]
@rubensworks I have updated it now. Please have a look.

**Comment by rubensworks** [comment]
Thanks, released as 1.5.0.

**Pull request merged**